components/cards/Cards.module.css

```
@@ -0,0 +1,52 @@
+.recipecard {
```

As you are using the `Cards.module.css` it is already clear that you are describing classes for the cards component.
```suggestion
.recipe {
```

---

would be fixed in next pullrequest. thank you